src/sentry/tasks/data_export.py

```diff
@@ -18,17 +20,21 @@ class DataExportError(Exception):
 
 
 @instrumented_task(name="sentry.tasks.data_export.assemble_download", queue="data_export")
-def assemble_download(data_export):
+def assemble_download(data_export_id):
+    # Extract the ExportedData object
+    try:
+        data_export = ExportedData.objects.get(id=data_export_id)
+    except ExportedData.DoesNotExist as error:
+        return capture_exception(error)
```

Comment:
@leeandher I actually think it would be better to use `logger.error` here so it goes to both Sentry and Kibana.

Reply:
I'm sending the raw error to Sentry, but in a future PR I'm adding the logger to send some custom messages to Kibana

Reply:
Wherever the `capture_exception` is currently, it shouldn't ever get to, so I just want to send the entire thing to Sentry. Later, in Kibana I can log that a failure happened but I dont know if we usually send them the unknown/unexpected errors.